src/components/Header/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    text-align: center;
+  }
```

Comment:
nevím jestli si o tom s luďkem ještě budeme povídat (=jestli je to v osnově), ale dej si bacha na to že i když css soubor importujeme v komponentě (Header/index.js) tak se pravidla v něm vztahují na celou stránku/aplikaci. Tj tímhle styluješ všechny nadpisy h2 ve všech komponentách v celé aplikaci. Proto v těch příkladech před a po luděk boužívá bem pojmenování tříd (movie__detail, movie__title, ...).